infra/resources/dev/templates/config.yaml

```diff
@@ -0,0 +1,49 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+
+processors:
```

Comment:
can you add a link to documentation?

Reply:
sorry I added this folder for a test, but it's not needed 😆  now I'll delete it